tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -0,0 +1,215 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.services.background_thread import BackgroundThreadService
+
+from kafkatest.services.kafka.directory import kafka_dir, KAFKA_TRUNK
+from kafkatest.services.kafka.version import TRUNK
+from kafkatest.services.security.security_config import SecurityConfig
+
+from collections import namedtuple
+import json
+import os
+import subprocess
+import time
+import signal
+
+TopicPartition = namedtuple('TopicPartition', ['topic', 'partition'])
```

Comment:
Though this is small, it's generally useful - maybe put it in in a file underneath `services/kafka/` and importable thusly
`from services.kafka import TopicPartition`?